scripts/gha/print_matrix_configuration.py

```diff
@@ -113,6 +113,17 @@
       "ndk_version": ["r22b"],
       "platform_version": ["28"],
       "build_tools_version": ["28.0.3"],
+
+      EXPANDED_KEY: {
+        # architecture_windows_linux, ndk_version, msvc_runtime: not enabled yet
+        "architecture_windows_linux": ["x64", "x86"],
```

Comment:
Since I think we're not currently testing on x86, if this fails, feel free to remove it until we can fix it.